### DISCUSSION TITLE
[Incorrect behavior][3.0.0-beta.2.6]: Custom MP4 intro movie does not play

### DISCUSSION THREAD

**Discussion by SorcererPonse** [Raytwo/ARCropolis#300] [Issues] [answered]
### Have you read the disclaimer before proceeding?

- [X] I have read the disclaimer.

### System Firmware Version

13.1.0

### Atmosphere Version

1.2.5

### Smash Version

13.0.1

### ARCropolis Version

3.0.0-beta.2.6

### Version channel

Beta

### SD card formatting

FAT32

### Can this incorrect behavior be consistently reproduced?

Yes

### Expected behavior

Place a custom c2_opening.mp4 in the /stream;/movie directory in a folder that ARC can discover, then start the game. The custom intro movie should play where the vanilla opening would normally play.

### Context

Starting the game with other mods enabled and no conflicts detected by ARCropolis. Nothing else unique was going on at the time.

### Configuration

version = '3.0.0-beta.2.6'
debug = false
auto_update = true
beta_updates = true
no_web_menus = false
region = 'us_en'

[paths]
arc = 'sd:/ultimate/Default/arc'
umm = 'sd:/ultimate/mods'
extra_paths = []

[logger]
logger_level = 'Off'
log_to_file = false

### Minimal modpack to observe the issue

https://gamebanana.com/mods/331864

**Comment by RelyksZv** [comment]
Can second this.

Edit. The stable version 2.1.1 works fine, however.

**Comment by Raytwo** [comment]
> Can second this.
> 
> Edit. The stable version 2.1.1 works fine, however.

Ok well, super late on this, apologies, but judging by this message I'll assume the issue is fixed. Thanks for reporting it!

**Comment by RelyksZv** [comment]
> > Can second this.
> > Edit. The stable version 2.1.1 works fine, however.
> 
> Ok well, super late on this, apologies, but judging by this message I'll assume the issue is fixed. Thanks for reporting it!

It still seems to be an issue on the most recent build

**Comment by Raytwo** [comment, marked as answer]
Took us a long time, but the next update (3.4.1) addresses this issue, so I'll be closing this now. Thank you for reporting it!

**Reply by RelyksZv** [reply]
Awesome! Looking forward to the future update. Thanks!!
